calcs/cpa_tcpa.js

```diff
@@ -121,6 +127,38 @@ module.exports = function (app, plugin) {
             },
             { latitude: vesselPos.latitude, longitude: vesselPos.longitude }
           )
+          
+          if (
+            typeof plugin.properties.traffic.distanceToSelf === 'undefined' ||
+            plugin.properties.traffic.distanceToSelf
+         ) {
+            app.debug('distance of ' + vessel + ' to self: ' + distance)
+            app.handleMessage(plugin.id, {
+              context: 'vessels.' + vessel,
+              updates: [
+                {
+                  values: [
+                    {
+                      path: 'distanceToSelf',
+                      value: distance
+                    }
+                  ],
+                  meta: [
```

Comment:
Sending meta on each update bloats them needlessly. Once should suffice - the server will make the meta available for downstream apps.

Reply:
removed line 132 (enabled calc by default) and deleted meta from update.